app/src/androidTest/java/com/google/android/stardroid/test/control/AstronomerModelTest.java

```diff
@@ -110,11 +110,9 @@ public void testSetPhoneSensorValues_phoneFlatAtLat0Long90() {
     Vector3 expectedEast = new Vector3(-1, 0, 0);
     Vector3 expectedSouth = new Vector3(0, 0, -1);
     Vector3 expectedWest = new Vector3(1, 0, 0);
-    Vector3 expectedPointing = expectedNadir;
```

Comment:
I'd like to keep these as they are. The variable names are meant to document their meaning.  In the case of this particular test we're showing that the "pointing" (ie the direction we're looking) is in the same direction as the nadir.

Reply:
(same elsewhere)